homeassistant/components/screenlogic/__init__.py

```diff
@@ -113,9 +114,8 @@ class ScreenlogicDataUpdateCoordinator(DataUpdateCoordinator):
 
     def __init__(self, hass, *, config_entry, gateway):
```

Comment:
```suggestion
    def __init__(self, hass: HomeAssistant, *: Any, config_entry: ConfigEntry, gateway: ScreenLogicGateway) -> None:
```

Than you can drop the typing on `self.config_entry` and `self.gateway` below since it will already be known

Reply:
Good call. Thanks.

Reply:
FYI, it doesn't like me trying to type hit `*: Any`.